--- CMakeLists.txt   2024-12-05 13:19:00
+++ CMakeLists.txt  2024-12-05 13:19:00
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.0)
 project(
   fatx
-  VERSION 1.17
+  VERSION 1.18
   LANGUAGES C CXX)
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR})
 if(CMAKE_BUILD_TYPE STREQUAL "")

--- CMakeLists.txt  2024-12-05 13:19:00
+++ CMakeLists.txt  2024-12-05 13:19:00
@@ -32,5 +32,5 @@
  add_compile_options(
   -D_FILE_OFFSET_BITS=64
-  -std=c++2b
+  -std=c++20
   -fdiagnostics-color
   -Werror

--- CMakeLists.txt  2024-12-05 13:19:00
+++ CMakeLists.txt  2024-12-05 13:19:00
@@ -192,11 +192,11 @@
    ${i} ALL
    COMMAND ${CMAKE_COMMAND} -E create_symlink fatx ${i}
    DEPENDS fatx)
-install(FILES ${PROJECT_BINARY_DIR}/${i} TYPE SBIN)
+install(FILES ${PROJECT_BINARY_DIR}/${i} TYPE BIN)
 endforeach(i)

 include(GNUInstallDirs)
-install(TARGETS fatx RUNTIME DESTINATION sbin)
+install(TARGETS fatx RUNTIME DESTINATION bin)
 install(DIRECTORY man8 TYPE MAN)
 install(
   FILES README

--- src/fatx.hpp    2024-12-05 13:19:00
+++ src/fatx.hpp    2024-12-05 13:19:00
@@ -56,13 +56,14 @@
-#include <fstream>
-#include <iostream>
-#include <filesystem>
-#include <thread>
-#include <string>
-#include <vector>
-#include <memory>
-#include <map>
-#include <set>
-#include <cassert>
-#include <functional>
-#include <cstdint>
-#include <shared_mutex>
+#include <bitset>
+#include <cassert>
+#include <cstdint>
+#include <filesystem>
+#include <fstream>
+#include <functional>
+#include <iostream>
+#include <map>
+#include <memory>
+#include <set>
+#include <shared_mutex>
+#include <string>
+#include <thread>
+#include <vector>
